algorithm-prototyping/brute-force-prototype/src/test/java/ScheduleUtilTest.java

```diff
@@ -97,15 +101,23 @@ static void setUp() {
         invalidTestSchedules.add(new Schedule(Set.of(
                 courses.get("ENGL 1102").sections().get(1),
                 courses.get("CSCI 1302").sections().get(2),
-                courses.get("MATH 2250").sections().get(1),
-                )));
+                courses.get("MATH 2250").sections().get(1)
+        )));
     }
 
     @Test
     void testValidate() {
         // Assert that ScheduleUtil.validate() returns true for all schedules in validTestSchedules
         // and returns false for all schedules in invalidTestSchedules
-    }
+        for (Schedule schedule : validTestSchedules) {
+            // should print true 5 times
+            System.out.println(ScheduleUtil.validate(schedule));
+        } // for
+        for (Schedule schedule : invalidTestSchedules) {
+            // should print false 5 times
+            System.out.println(ScheduleUtil.validate(schedule));
+        } // for
```

Comment:
For future reference, these print statements should be replaced with `assertTrue` and `assertFalse`. For the sake of time, I will just merge this and make these changes myself.

Reply:
Also, avoid writing "// for" and "// testValidate". I understand it's part of the style guide for CS1302, but it's unnecessary and dated when using an IDE.